index.d.ts

```diff
@@ -244,4 +339,4 @@ declare module 'binance-api-node' {
         isOrderWorking: boolean;
         isBuyerMaker: boolean;
     }
-}
+}
```

Comment:
looks good, can you just keep the ending newline?

Reply:
Done